tunnel/intercept/interceptor.go

```diff
@@ -72,6 +73,11 @@ func (addr *InterceptAddress) Contains(ip net.IP, port uint16) bool {
 	return addr.cidr.Contains(ip) && port >= addr.lowPort && port <= addr.highPort
 }
 
+func (addr *InterceptAddress) String() string {
+	return fmt.Sprintf("cidr: %v, cidrAddr: %p, lowPort: %v, highPort: %v, protocol: %v, tproxySpec: %v, acceptSpec: %v",
```

Comment:
should this have a nil guard?

Reply:
For if the addr is null? Shouldn't ever called on a nil addr. It's only used for debugging what's in the list of addresses and the list of addresses should never contain nils